src/bullet/index.tsx

```diff
@@ -37,6 +38,11 @@ const StyledBullet = styled(Bullet)`
     border-color: ${color.polylineSecondary};
     box-shadow: 0 0 0 1px ${color.polylineStrokeSecondary};
   }
+
+  &.kirk-bullet--search {
+    border-width: 4px;
+    border-color: ${color.secondaryText} !important;
```

Comment:
`!important`??!

Reply:
Aaaah this is terrible!
I used it for a test and forgot to remove it :(